### DISCUSSION TITLE
Is it possible to make an entry without title?

### DISCUSSION THREAD

**Discussion by MW3000** [jrnl-org/jrnl#1670] [Q&A] [answered]
I want most of my entries without title. I use jrnl just by calling the command `jrnl`.
Is it possible to make an entry without title?

**Comment by wren** [comment, marked as answer]
Hi, @MW3000 

I think this is an interesting idea. Right now, the title is required for all entries. Right now, technically, the "title" is the actual entry, and the "body" is optional.

That being said, we're looking at updating `jrnl` into a new file format (see #1610). One of the new concepts we are planning to introduce is metadata. So, given your question, we could turn the title field into metadata, and have it be optional, which would allow you to make entries with no title.

**Reply by MW3000** [reply]
Yes, to me it would make sense to have the title optional. As we are used to it by micro blogging systems. I have also thought about only using the title. But it would be nice to be able to have entries with or without a title.

**Comment by wren** [comment]
I'm looking at this closer, and I think a workaround for this would be to place a period as the first line in your editor. Then the period will be the title for all of your entries. This sort of gets you entries with no title in the current format.

@MW3000 Do you think that would work for you?

**Reply by MW3000** [reply]
Yes, thank you, @wren. That's what I am doing right now. It is good enough as a workaround.
